src/ZendeskApi.Contracts/Models/Organization.cs

```diff
@@ -49,8 +49,8 @@ public class Organization : IZendeskEntity
         [IgnoreDataMember]
         public long group_id { get; set; }
 
-        [IgnoreDataMember]
-        public long external_id { get; set; }
+        [DataMember(Name = "external_id", EmitDefaultValue = false)]
```

Comment:
Hello,

i've just commited this change to my branch. As i told you i'm totally a newbie on Github and Git, i synched my changes to my repository. i need to do something else?